Tests/TensorFlowTests/OperatorTests/LinearAlgebraTests.swift

```diff
@@ -111,13 +111,78 @@ final class LinearAlgebraTests: XCTestCase {
         let computedGradient = gradient(at: input) { logdet($0).sum() }
         assertEqual(computedGradient, expectedGradient, accuracy: 1e-5)
     }
+    
+    func testTriangularSolve() {
+        let a = Tensor<Float>([
+            [1, 0, 0],
+            [2, 1, 0],
+            [3, 2, 1]
+        ])
+        let y = Tensor<Float>([1, 1, 3]).reshaped(to: [-1, 1])
+        let y2 = Tensor<Float>([y, y])
+        let x = triangularSolve(matrix: a, rhs: y2)
+    }
+    
+    /// Value of the gradient was computed using the following code:
```

Comment:
I think it would be better if you could shift the python TensorFlow 2.0 code within the test itself.

Reply:
@Shashi456, yes, I do that but for different shapes to check that broadcasting in triangular solve works correctly.